brave/brave-ios
# Fix #2999: Add new onboarding for setting up default browser.

## Summary of Changes


This pull request fixes #2999 

## Submitter Checklist:


## Test Plan:

General logic goes like this:
We show it for every user on second app launch,
then we show it again after 5 days(or 5 minutes on dev)
then we do not show this callout anymore

1. New user
- Fresh install, open the app once
- Kill the app and reopen
- Verify that default browser callout is showing
- Kill the app, wait 5minutes+(on dev build)
- Verify that popup shows again

1. Existing user
- Upgrae
- Kill the app and reopen
- Verify that default browser callout is showing
- Kill the app, wait 5minutes+(on dev build)
- Verify that popup shows again


## Screenshots:



<img width="520" alt="Zrzut ekranu 2021-01-28 o 12 07 37" src="https://user-images.githubusercontent.com/6950387/106130827-2297e000-6162-11eb-8183-fb9dbf8c6903.png">


## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `release-notes/(include|exclude)`
  - `bug` / `enhancement`
